cfgov/unprocessed/js/organisms/FilterableListControls.js

```diff
@@ -72,7 +77,52 @@ function FilterableListControls( element ) {
    * Initialize FilterableListControls events.
    */
   function _initEvents() {
-    _form.addEventListener( 'submit', _formSubmitted );
+    var labelDom = _dom.querySelector( '.o-expandable_label' );
+    var label;
+
+    if ( labelDom ) {
+      label = labelDom.textContent.trim();
+    }
+
+    _expandable.addEventListener( 'expandBegin',
+    function sendEvent( ) {
+      Analytics.sendEvent( 'Filter:open', label );
+    } );
+
+    _expandable.addEventListener( 'collapseBegin',
+    function sendEvent( ) {
+      Analytics.sendEvent( 'Filter:close', label );
+    } );
+
+    _form.addEventListener( 'change', function sendEvent( event ) {
+      var action;
+      var field = event.target;
+      var fieldValue;
+      var fieldLabel;
+
+      if ( !field ) {
+        return;
+      } else if ( !_cachedLabels[field.name] ) {
+        _cachedLabels[field.name] = {
+          label:  _getLabelText( field )
+        };
+      }
+
+      if ( field.type === 'checkbox' && field.checked === false ) {
+        fieldValue = 'null';
+      } else {
+        fieldValue = field.value;
+      }
+
+      fieldLabel = _cachedLabels[field.name].label + ':' + fieldValue;
+      action = 'Filter:' + field.type + ':changed';
```

Comment:
> Did the action and event naming conventions come from the analytics team?


I kind of piggybacked on what the eRegs team was doing :https://github.com/cfpb/regulations-site/blob/84891b05e8c24530b78260593bfca29e5b44ecb8/regulations/static/regulations/js/source/views/drawer/search-view.js#L28